ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManager.java

```diff
@@ -197,7 +197,9 @@ public HelixClusterManager(ClusterMapConfig clusterMapConfig, String instanceNam
   private HelixManager initializeHelixManagerAndPropertyStoreInLocalDC(Map<String, DcZkInfo> dataCenterToZkAddress,
       String instanceName, HelixFactory helixFactory) throws Exception {
     DcZkInfo dcZkInfo = dataCenterToZkAddress.get(clusterMapConfig.clusterMapDatacenterName);
-    String zkConnectStr = dcZkInfo.getZkConnectStr();
+    // For now, the first ZK endpoint (if there are more than one endpoints) will be adopted by default. We can update
+    // this if we really need to initialize mulitiple HelixManager in local dc.
+    String zkConnectStr = dcZkInfo.getZkConnectStrs().get(0);
```

Comment:
this needs to go below line 203 since CLOUD_BACKED replicas do not have a zk connect string